Old Projects Added & Current Projects Cleanup

Grallistrix opened the pull request:
Added some old projects that i found (ADR,EVE,KRAK,CAR) - made their articles, added logos and gallery
Small clean up of current projects - added lost logos, some photos to galleries, justified their text
Current projects index - fixed some broken links, now DuckieTown and TWT articles can be visited

Grallistrix committed:
New Old Photos

Added new photos for:
-TWT
-Kraksat
-ADR
-Duckie
-CarMODEL
-EVE 1.0
-Micromouse Family

Grallistrix committed:
The Graveyard Update

Added the finished projects:
ADR
CarMODEL
EVE
KrakSat

Grallistrix committed:
The Graveyard Update 2 - Now with Photos

Added logos and galleries for finished projects:
ADR
EVE
CarMODEL

Also - updated the index page for finished projects
KrakSat

Grallistrix committed:
Updated DuckieTownLogo

Updated to a transparent one without the white outline

Grallistrix committed:
New Old Photos 2

Addes some more old photos

Grallistrix committed:
Even More Photos

Grallistrix committed:
DuckiePhoto

Grallistrix committed:
The Just and Beautiful Update

Justified the text in all projects
Added new photos and galleries to them
Fixed some broken links in idex

iwo-strzebonski commented on content/blog/projects/current/duckie-town.md:
```diff
@@ -14,12 +14,29 @@ head:
-# Duckie Town
 
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/8e8e1439-f72e-44a5-9795-7f91a1220264" alt="DuckieTOWN Logo" style="max-height: 500px;  display: block; margin: 0; padding-left:60px">
```
please use Markdown tags for inserting images

iwo-strzebonski commented on content/blog/projects/current/duckie-town.md:
```diff
@@ -14,12 +14,29 @@ head:
       content: 2023-09-20T12:23:00Z
 ---
 
-# Duckie Town
```
`lastmod` field wasn't changed

iwo-strzebonski commented on content/blog/projects/current/f1-10.md:
```diff
@@ -15,14 +15,15 @@ head:
 
 # F1TENTH
-
+<p style="text-align:justify">
```
As every blank-spaced block of text is already a paragraph, instead of putting whole text in another paragraph, request adding the following in `integra.front` stylesheet:

<img width="152" alt="image" src="https://github.com/KN-Integra/integra.content/assets/61759168/b325f291-a296-41f0-92b9-1a777774a6da">

Also request that `<a>` tags should be `inline` instead of `inline-block` for content pages.

iwo-strzebonski commented on content/blog/projects/current/lf-hammer.md:
```diff
@@ -16,6 +16,8 @@ head:
 
 # Line Follower Hammerhead
 
+<p style = "text-align: justify">
```
See comment for F1TENTH

iwo-strzebonski commented on content/blog/projects/current/twt.md:
```diff
@@ -14,6 +14,10 @@ head:
 ---
 
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/35fa5843-3b80-4ceb-83df-ea8f35835f84" alt="TWT Logo" style="max-height: 300px;  display: block; margin: 0; padding-left:140px">
```
Use correct Markdown syntax and change the lastmod date.
If exact sizes are required, request a new component in the `integra.front` repo

iwo-strzebonski commented on content/blog/projects/current/wyndka.md:
```diff
@@ -15,12 +15,16 @@ head:
 
 # Wędka Wspomagana Elektrycznie "WWE Sumołów 2000"
+
+<p style="text-align:justify">
```
See comment for F1TENTH

iwo-strzebonski commented on content/blog/projects/current/zebulon.md:
```diff
@@ -14,14 +14,20 @@ head:
 
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/445f7e75-925e-4744-995f-e778741361c4" alt="Image description" style="max-height: 500px;  display: block; margin: 0;">
```
Use correct Markdown syntax and change the lastmod date.
If exact sizes are required, request a new component in the `integra.front` repo

iwo-strzebonski commented on content/blog/projects/current/zebulon.md:
```diff
@@ -14,14 +14,20 @@ head:
+
 # Zebulon i Zebulon 2.0
 
+<p style="text-align:justify">
```
See comment for F1TENTH

iwo-strzebonski commented on content/blog/projects/finished/adr.md:
```diff
@@ -0,0 +1,41 @@
+      content: 2023-03-20T19:40:00Z
+---
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/e6f8639b-19d4-450a-836b-1aa75fe1f474" alt="Image description" style="max-height: 250px;  display: block; margin: 0; padding-left:160px">
```
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo

iwo-strzebonski commented on content/blog/projects/finished/carmodel.md:
```diff
@@ -0,0 +1,46 @@
+
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/439fe2cf-ea08-42c0-b7b0-92a9ee60a5eb" alt="CarMODEL Logo" style="max-height: 300px;  display: block; margin: 0; padding-left:180px">
```
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo

iwo-strzebonski commented on content/blog/projects/finished/eve.md:
```diff
@@ -0,0 +1,28 @@
+</p>
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/3353e00f-aeea-404c-b0f5-fc5997ce6510" alt="Image description" style="max-height: 650px;  display: block; margin: 0;">
```
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo

iwo-strzebonski commented on content/blog/projects/finished/kraksat.md:
```diff
@@ -0,0 +1,36 @@
+
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/9966c763-d651-483e-b9c4-ba40aa66b6c2" alt="Logo KRAKSAT" style="max-height: 350px;  display: block; margin: 0; padding-left: 120px">
```
Use correct Markdown syntax.
If exact sizes are required, request a new component in the `integra.front` repo

iwo-strzebonski commented on content/blog/projects/finished/adr.md:
```diff
@@ -0,0 +1,41 @@
+</p>
+
+# Autonomus Delivery Robot
+<p style="text-align:justify">
```
See comment for F1TENTH

iwo-strzebonski commented on content/blog/projects/finished/carmodel.md:
```diff
@@ -0,0 +1,46 @@
+
+# CarMODEL
+
+<p style="text-align:justify">
```
See comment for F1TENTH

iwo-strzebonski commented on content/blog/projects/finished/eve.md:
```diff
@@ -0,0 +1,28 @@
+---
+
+# EVE
+<p style="text-align:justify">
```
See comment for F1TENTH

iwo-strzebonski commented on content/blog/projects/finished/index.md:
```diff
@@ -16,4 +16,14 @@ head:
 
 # Zakończone projekty
 
-## [Projekt 1](/projects/finished/project1)
```
Please change the lastmod date

iwo-strzebonski commented on content/blog/projects/finished/kraksat.md:
```diff
@@ -0,0 +1,36 @@
+</a>
+
+# KrakSat
+<p style="text-align: justify">
```
See comment for F1TENTH

Grallistrix committed:
Image Componend impelemtation - Current

changed previous html code images into their /component/image implementation for current projects

Grallistrix committed:
Image component implementation - fiished projects

implemented the image component for the finished porjects
